.github/workflows/test-js-packages.yml

```diff
@@ -557,11 +555,17 @@ jobs:
           - project: { repo: AztecProtocol/aztec-packages, path: noir-projects/noir-protocol-circuits/crates/parity-lib }
           - project: { repo: AztecProtocol/aztec-packages, path: noir-projects/noir-protocol-circuits/crates/private-kernel-lib }
           - project: { repo: AztecProtocol/aztec-packages, path: noir-projects/noir-protocol-circuits/crates/reset-kernel-lib }
-          - project: { repo: AztecProtocol/aztec-packages, path: noir-projects/noir-protocol-circuits/crates/rollup-lib }
           - project: { repo: AztecProtocol/aztec-packages, path: noir-projects/noir-protocol-circuits/crates/types }
+          # This one currently doesn't compile (it runs out of memory or it takes too long)
+          # - project: { repo: AztecProtocol/aztec-packages, path: noir-projects/noir-protocol-circuits/crates/rollup-lib, nargo_args: "--test-threads 1" }
```

Comment:
Does this fail in a reasonable amount of time? If so can we reenable this so we can track how it fails (and when it becomes to pass again)?

Reply:
Around 4 minutes. I'll try re-enabling it and removing/renaming the corresponding file. I think the big curve one takes more time than this.

Reply:
I think the problem is that GitHub cancels the job and it fails. It doesn't even reach the comparison step. So I'll undo that commit (but feel free to try something else).